developers/weaviate/starter-guides/compression.mdx

```diff
@@ -0,0 +1,149 @@
+
+SQ and BQ both have optional vector caches. Use these configurable caches to load frequently used, uncompressed vectors into memory to improve overall search times.
+
+#### Import speed
+
+Importing and compressing vectors takes slightly longer than just importing them, but this is a one time cost. In contrast, loading a compressed index into memory is faster since there is less data to load.
+
+Starting in v1.22, Weaviate has an optional, [asynchronous indexing](/developers/weaviate/config-refs/schema/vector-index#asynchronous-indexing) feature which improves the import process. When asynchronous indexing is enabled, indexing takes place as a background process. You can begin other tasks without having to wait for the entire import to be indexed. This means you are able to begin other tasks sooner.
+
+### Activate compression
+
+PQ and SQ both require training data. PQ has to define centroids for each segment. SQ has to determine the minimum and maximum values for the bucket boundaries. When you have imported a large enough training set, the algorithm will compress your data.
+
+SQ has to be enabled when you create the collection.
+
+If you have async indexing and [AutoPQ enabled](/developers/weaviate/configuration/compression/pq-compression#configure-autopq), PQ compression can be started anytime. If not, you should only enable PQ after you have imported enough objects to [train the algorithm](/developers/weaviate/configuration/compression/pq-compression#manually-configure-pq).
+
+BQ doesn't require a training step, however it can only be enabled when you create your collection. BQ cannot be enabled after you start to add data to the collection.
+
+## Recommendations
+
+Most applications benefit from compression.
+
+- The cost savings are significant. In [Weaviate Cloud](https://weaviate.io/pricing), for example, uncompressed collections can be 5 to 8 times as expensive as compressed collections.
+- If you have a small collection that uses a flat index, consider a BQ index. The BQ index is 32 times smaller and much, much faster then the uncompressed equivalent.
+- If you have specialized needs and a very large data set, consider PQ compression. PQ compression is very configurable, but it requires more expertise to tune well than SQ or BQ.
+- Most users with medium to large data sets should consider SQ compression. SQ compressed vectors are one quarter the size of uncompressed vectors. Searches with SQ are several times faster than searches with uncompressed vectors. Recall is similar to uncompressed data.
+
+Finally, for collections that are small, but that are expected to grow, consider a dynamic index. Configure the collection to use BQ compression while the index is flat and SQ compression when the collection grow large enough to move from a flat index to an HNSW index.
+
+## Related pages
+
+For more information on compression, see these pages.
+
+### Documentation pages
```

Comment:
Let's add something here to say that you can go to these pages to learn how to configure/enable these compressions